packages/bpk-docs/src/pages/NativeNudgerPage/NativeNudgerPage.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import readme from 'react-native-bpk-component-nudger/readme.md';
+
+import iosScreenshotDefault from 'react-native-bpk-component-nudger/screenshots/ios/default.png';
+import androidScreenshotDefault from 'react-native-bpk-component-nudger/screenshots/android/default.png';
+
+import Paragraph from './../../components/Paragraph';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb:
+      'Nudgers come with decrease and increase buttons to either side of the selected value. A minimum and maximum value need to be provided.',
```

Comment:
Might sound better to say "on either side"

Reply:
Good plan. I copied this from the web nudger page, so I'll change it in both places.